source/Jobbr.Server.UnitTests/Storage/InMemoryJobStorageProviderTests.cs

```diff
@@ -0,0 +1,412 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Jobbr.ComponentModel.JobStorage;
+using Jobbr.ComponentModel.JobStorage.Model;
+using Jobbr.Server.Storage;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Shouldly;
+
+namespace Jobbr.Server.UnitTests.Storage
+{
+    [TestClass]
+    public class InMemoryJobStorageProviderTests
+    {
+        private readonly IJobStorageProvider _provider;
+
+        public InMemoryJobStorageProviderTests()
+        {
+            _provider = new InMemoryJobStorageProvider();
+        }
+
+        [TestMethod]
+        public void AddJob_ShouldSetId()
+        {
+            // Arrange
+            var jobs = Enumerable.Range(0, 4).Select(_ => new Job()).ToList();
+
+            // Act
+            jobs.ForEach(job => _provider.AddJob(job));
+
+            // Assert
+            jobs.ShouldAllBe(job => job.Id != 0);
+            jobs.Select(job => job.Id).ShouldBeUnique();
+            _provider.GetJobsCount().ShouldBe(jobs.Count);
+        }
+
+        [Ignore]
```

Comment:
Why ignore?

Reply:
Can you add a justification to the attribute, so it's clearer: `[Ignore("blabla")]`